docs/source/index.rst

```diff
@@ -18,34 +18,3 @@ Guide
    help
    doc_maintenance
 
-
-Tutorial Documents
-==================
-
-Tutorial documents provide specific instructions to help you get started.
-
-
-How-To Documents
-================
-
-How-To documents explain how to accomplish specific tasks.
-
-
-Reference Documents
-===================
-
-Reference documents describe application concepts in depth.
-
-
-Discussion Documents
-====================
-
-Discussion documents provide insights into integration of the application into your own software development life cycle.
-
-
-Indices and Tables
-==================
-
-* :ref:`genindex`
-* :ref:`modindex`
-* :ref:`search`
```

Comment:
I think we don't need these if we are doing in the way it is.

Reply:
Hi, we were thinking about keeping these "Tutorials", "How To", "Reference" and  "Discussion" headings as that would mean someone can directly go to Tutorial documents only. These headings would have AboutCode projects like scancode-toolkit, scancode-workbench as sub-headings and the corresponding Tutorial/Discussion/HowTo/Explanation files under those project headings. We could delete the `search` though because it already exists.